src/ontology/edits/oeo-model.omn

```diff
@@ -3894,6 +3804,31 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1990",
         <http://purl.obolibrary.org/obo/BFO_0000002>
     
     
+Class: OEO_00410009
+
+    Annotations: 
+        OEO_00020426 "issue: https://github.com/OpenEnergyPlatform/ontology/issues/1962
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1991",
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A legislation is a plan specification that is produced by a government and enables, prohibits or bans activities, or establishes rights or responsibilities."@en,
```

Comment:
We used http://addictovocab.org/ADDICTO_0000488 as inspiration for the definition. Please add as definition source a comment "Based on http://addictovocab.org/ADDICTO_0000488" or so...